gprofiler/metadata/application_identifiers.py

```diff
@@ -273,6 +278,17 @@ def get_app_id(self, process: Process) -> Optional[str]:
         return None
 
 
+class _NodeModuleApplicationIdentifier(_ApplicationIdentifier):
+    def get_app_id(self, process: Process) -> Optional[str]:
+        if not _is_node_bin(_get_cli_arg_by_index(process.cmdline(), 0)):
+            return None
+
+        arg_1 = _get_cli_arg_by_index(process.cmdline(), 1)
```

Comment:
The command line can be:
* `node myapp.js`
* `node --some-flag myapp.js`
* `node -r myrequire.js --some-flag myapp.js`

So you need a more sophisticated way to find the arg than simply getting the first one. Please also write tests for the cases I suggested